misc-utils/getopt.c

```diff
@@ -126,53 +129,40 @@ static void print_normalized(const struct getopt_control *ctl, const char *arg)
 	 * and an opening quote! We need also the global opening and closing
 	 * quote, and one extra character for '\0'.
 	 */
-	buf = xmalloc(strlen(arg) * 4 + 3);
-	bufptr = buf;
+	buf = xmalloc(strlen(arg) * 4 + 1);
 
```

Comment:
Why we need this change? .. due to final the printf() at the end of the function, right?